pkg/apis/application/v1alpha1/types.go

```diff
@@ -2427,12 +2427,18 @@ func (s *SyncWindows) inactiveAllows(currentTime time.Time) *SyncWindows {
 		for _, w := range *s {
 			if w.Kind == "allow" {
 				schedule, sErr := specParser.Parse(w.Schedule)
+				if sErr != nil {
+					continue
```

Comment:
```suggestion
                                        log.Warnf("invalid schedule '%v': %s", w.Schedule, err)
					continue
```

Reply:
Thanks for the comment! I've added the log.